internal/config/options.go

```diff
@@ -208,6 +210,7 @@ func OptionsFromViper(configFile string) (*Options, error) {
 		return nil, err
 	}
 
+	log.Debug().Msgf("Read configuration with checksum: %s", o.Checksum())
```

Comment:
See above comment about preferring structured logging vs Msgf

Reply:
Should be addressed.  See if it looks good now.